Mention the existence of the CURRENT key file (#12734)
DataDog/documentation: content/en/agent/faq/linux-agent-2022-key-rotation.md

package gpg-pubkey-fd4bf915 is not installed
```

> Alternatively, you can check if you `datadog.repo` file contains https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public as one of the `gpgkey` entries. This key file will be updated with the new key as soon as it is in use.

Reviewer: ```suggestion
Alternatively, check if your `datadog.repo` file contains `https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public` as one of the `gpgkey` entries. This key file is updated with the new key as soon as it is in use.
```

Author: I still think we should say "will be updated" instead of "is updated" since this will be a one-shot thing that will happen in June this year (and we are asking customers to be prepared for it).

> Alternatively, you can check if you `datadog.repo` file contains https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public as one of the `gpgkey` entries. This key file will be updated with the new key as soon as it is rotated.

Reviewer 2: To be more precise, should this be "This key file will be updated with the new key as soon as the repository metadata gets signed with the new key."?

Author: ```suggestion
Alternatively, you can check if you `datadog.repo` file contains https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public as one of the `gpgkey` entries. This key file will be updated with the new key as soon as it is in use.
```
What about "as soon as it is in use"? Since we don't clarify the difference between signing the metadata vs the actual packages anywhere in these docs.

Reviewer 2: Yeah, I think that's good 👍
